src/main/java/com/sap/cloudfoundry/client/facade/util/Messages.java

```diff
@@ -0,0 +1,7 @@
+package com.sap.cloudfoundry.client.facade.util;
+
+public class Messages {
+
+    // ERROR messages
+    public static final String COULD_NOT_RETRIEVE_CLOUD_STACK = "Could not retrieve cloud stack by id.";
```

Comment:
A "cloud stack" doesn't exist. Just "stack"s do. Prefer not to use our custom implementation names for CF entities.